pycyphal/transport/udp/_session/_output.py

```diff
@@ -180,11 +182,13 @@ async def _emit(
         for index, (header, payload) in enumerate(header_payload_pairs):
             try:
                 # TODO: concatenation is inefficient. Use vectorized IO via sendmsg() instead!
+                _logger.debug("starting send: %s, using socket: %s", b"".join((header, payload)), self._sock)
```

Comment:
Every time you perform a complex computation just to emit a debug log message, god kills a kitten.

```suggestion
                combined_payload = b"".join((header, payload))
                _logger.debug("%s: sending: %s", self, combined_payload)
```